Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py

```diff
@@ -350,7 +351,41 @@ def testSpiroAndBridgeheads(self):
     self.assertEqual(len(sa),2)
     self.assertEqual(sorted(sa),[6,9])
     
-
-
+  def testNumRotatableBonds(self):
+    self.assertEquals(-1,
+           int(rdMD.NumRotatableBondsOptions.Default))
+    self.assertEquals(0,
+           int(rdMD.NumRotatableBondsOptions.NonStrict))
+    self.assertEquals(1,
+           int(rdMD.NumRotatableBondsOptions.Strict))
+    self.assertEquals(2,
```

Comment:
It's not really a problem with the PR, but are the integer values of these constants actually important?